src/bindings/python/src/pyopenvino/graph/any.cpp

```diff
@@ -62,6 +67,72 @@ void regclass_graph_Any(py::module m) {
     ov_any.def("__eq__", [](const ov::Any& a, py::object& b) -> bool {
         return a == ov::Any(Common::utils::py_object_to_any(b));
     });
+    ov_any.def(
+        "astype",
+        [](ov::Any& self, py::object dtype) {
+            if (check_key(dtype, py::bool_())) {
+                return py::cast(self.as<bool>());
+            } else if (check_key(dtype, py::str())) {
+                return py::cast(self.as<std::string>());
+            } else if (check_key(dtype, py::int_())) {
+                return py::cast(self.as<int64_t>());
+            } else if (check_key(dtype, py::float_())) {
+                return py::cast(self.as<double>());
+            } else if (check_key(dtype, py::dict())) {
+                return Common::utils::from_ov_any_map_no_leaves(self);
+            }
+            std::stringstream str;
+            str << "Unsupported data type : '" << dtype << "' is passed as an argument.";
+            OPENVINO_THROW(str.str());
+        },
+        R"(
+            Returns runtime attribute casted to defined data type.
+
+            :param dtype: Data type in which runtime attribute will be casted.
+            :type dtype: Union[bool, int, str, float, dict]
+
+            :return: A runtime attribute.
+            :rtype: Any
+    )");
+    ov_any.def(
+        "aslist",
+        [](ov::Any& self, py::object dtype) {
+            // before serialization
+            if (self.is<Common::utils::EmptyList>() || dtype.is_none()) {
+                return py::cast(std::vector<int>());
```

Comment:
There isn't default behavior in C++ for empty vectors which are expected from `get_rt_info(args)`.  
So according to [the test in c++ api](https://github.com/openvinotoolkit/openvino/blob/master/src/core/tests/pass/serialization/serialize.cpp#L432) user still should define vector of `float/string/something_else` is expected even for empty.

I decided to use `int` because it's a very basic type and in this case there no difference what type of vector will be.

Reply:
discussed offline -- decided to return python list.